packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -1521,4 +1527,67 @@ void main() {
     expect((mergeableMaterial.children.first as MaterialSlice).color, firstPanelColor);
     expect((mergeableMaterial.children.last as MaterialSlice).color, secondPanelColor);
   });
+
+  testWidgets('ExpansionPanelList icon color test', (WidgetTester tester) async {
+    const Color expandedIconColor = Colors.orange;
+    const Color iconColor = Colors.red;
+
+    await tester.pumpWidget(
+      const MaterialApp(
+        home: SingleChildScrollView(
+          child: SimpleExpansionPanelListTestWidget(
+            iconColor: iconColor,
+            expandedIconColor: expandedIconColor,
+          ),
+        ),
+      ),
+    );
+
+    IconButton iconButton = tester.widget(find.byType(IconButton).at(0));
```

Comment:
change `.at(0)` to `first` here and below 2 places too.

```suggestion
    IconButton iconButton = tester.widget(find.byType(IconButton).first);
```